### ISSUE TITLE
JSONObject["industry_stats"] not found

### ISSUE THREAD

**Issue by Schacal** [banana-j/bananaj#42]
Hello, 

I'm using your library in my project:
`<version>0.6.1</version> jdk 8`

I have a code:
```java
Campaign campaign = connection.createCampaign(CampaignType.REGULAR, new CampaignRecipients(segment), campaignSettings);
campaign.send();
Report report = campaign.getReport();
```

During the invocation of `campaign.getReport();` the error is thrown:
```java
org.json.JSONException: JSONObject["industry_stats"] not found.
	at org.json.JSONObject.get(JSONObject.java:520)
	at org.json.JSONObject.getJSONObject(JSONObject.java:732)
	at com.github.alexanderwe.bananaj.model.report.Report.<init>(Report.java:60)
	at com.github.alexanderwe.bananaj.model.campaign.Campaign.getReport(Campaign.java:219)
```
Could you please have a look on that issue?

**Comment by gscriver** [comment]
I have been unable to reproduce this issue. "industry_stats" always seems to be included in the report response. Perhaps this was a new account or the account stats are not configured? 
I'll make the "industry_stats" field optional.

**Comment by gscriver** [comment]
Fixed in build 0.6.2
